### ISSUE TITLE
Correct trigger fires only once

### ISSUE THREAD

**Issue by jukben** [webscopeio/react-textarea-autocomplete#145]
I think I have a similar problem:

I wanna use it in a project but there is an requirement named "Group mentions", which means besides the known "@user" autocomplete there is also an "@@group" feature.

So I have two triggers: "@" and "@@". 
My problem is that since "@" is triggered, the trigger of "@@" only fires once and than after another char never again. 

Here you can finde the example code: https://codesandbox.io/embed/8x9qww899l

_Originally posted by @Psyphant in https://github.com/webscopeio/react-textarea-autocomplete/issues/62#issuecomment-480375177_

**Comment by jukben** [comment]
Has been fixed in 4.3.4 (https://codesandbox.io/embed/9ljo56oz3w) 🚀

**Comment by OlegVach** [comment]
Hello guys. I have same problem but symbols is "{{". I'm using 4.5.0 version.

**Comment by jukben** [comment]
Hey @OlegVach could you please create a new issue with repro? 🙏 I'd very appreciate it!

**Comment by OlegVach** [comment]
@jukben It doesn't work with version 4.3.4. I will create new issue.
